Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/repository/storage/VaccinationStorage.kt

```diff
@@ -57,6 +58,22 @@ class VaccinationStorage @Inject constructor(
             }
         }
 
+    init {
+        // TODO REMOVE when all is done
+        if (CWADebug.isDebugBuildOrMode && !CWADebug.isAUnitTest) {
+            personContainers = setOf(
+                PersonData(
```

Comment:
`VaccinatedPersonData` ? we could have some person data somewhere else such RAT profile  too